collivery.php

```diff
@@ -1,18 +1,18 @@
 <?php
 
 define('_MDS_DIR_', __DIR__);
```

Comment:
Just as a side note, When Woocommerce jumps to version 4 this version should also jump

Reply:
Thanks